receiver/purefbreceiver/README.md

```diff
@@ -0,0 +1,53 @@
+# Pure Storage FlashBlade Receiver
+
+| Status                   |                     |
+| ------------------------ |---------------------|
+| Stability                | [in-development]    |
+| Supported pipeline types | metrics             |
+| Distributions            | [contrib]           |
+
+The Pure Storage FlashBlade receiver, receives metrics from Pure Storage FlashBlade via the [Pure Storage FlashBlade OpenMetrics Exporter](https://github.com/PureStorage-OpenConnect/pure-fb-openmetrics-exporter)
+
+Supported pipeline types: metrics
+
+## Configuration
+
+The following settings are required:
+ -  `endpoint` (default: `http://172.31.60.207:9491/metrics/array`): The URL of the scraper selected endpoint
+
+### Important 
+
+- Only endpoints explicitly added on will be scraped. e.g: `clients`
+
+Example:
+
+```yaml
+extensions:
+  bearertokenauth/fb01:
+    token: "..."
+
+receivers:
+  purefb:
+    endpoint: http://172.31.60.207:9491/metrics
+    arrays:
+      - address: fb01
+        auth:
+          authenticator: bearertokenauth/fb01
+    clients:
+      - address: fb01
+        auth:
+          authenticator: bearertokenauth/fb01
+    env: dev
+    settings:
+      reload_intervals:
+        array: 10s
```

Comment:
I think this should have the values closer to the ones users want. If the default is 5 minutes, this should be then, say, 6 minutes. Leaving it 10s kind of encourages users to use that value. You'd be surprised to know that most people will just copy and paste this config straight into their collector's :-)